modules/async-icq/testing/demo-simapp/README.md

```diff
@@ -0,0 +1,100 @@
+# interchainquerydemo
+**ICQ Demo** 
+
+## Disclaimer
+
+The following repository and [`x/interquery`](./x/interquery/) module serves as an example and is used to exercise the functionality of Interchain Accounts end-to-end for development purposes only.
+This module **SHOULD NOT** be used in production systems
+
+## Overview 
+
+The following repository contains a basic example of an Interchain Queries module and serves as a developer guide for teams that wish to use interchain queries functionality.
+
+### Developer Documentation
+
+Interchain Queries developer docs can be found here
+
+https://github.com/strangelove-ventures/ibc-go/blob/feature/icq_implementation/modules/apps/icq/README.md
```

Comment:
we could reference the updated doc in this repo now, https://github.com/cosmos/ibc-apps/tree/main/modules/async-icq

Reply:
053a49f9ef48e70b3f49267db7cf95d8935bf620